CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelogs
 
+## 0.16.0
+
+### Added
+
+- Support Pulumi: [#759](https://github.com/mingrammer/diagrams/pull/759)
+
```

Comment:
This change log file is outdated. We're using the [releases](https://github.com/mingrammer/diagrams/releases) to track the changes. So you don't have to update it. And I'll remove it soon.